test/mailers/volunteer_mailer_test.rb

```diff
@@ -25,4 +30,18 @@ def setup
     assert_match 'Bogus Education', mail_body
     assert_match 'Woman', mail_body
   end
+
+  test 'probation_period_mailer' do
+    _, _, group_assignments = create_group_offer_entity(
+      nil, 7.weeks.ago, nil, create(:volunteer_with_user), create(:volunteer_with_user)
+    )
+    assignment = make_assignment(start_date: 7.weeks.ago)
+    mailing = create_probation_mailing(*group_assignments, assignment)
+    mailing.reminder_mailing_volunteers.each do |rmv|
+      mailer = VolunteerMailer.probation_period_reminder(rmv).deliver
```

Comment:
I know you are using in the model `deliver_later` to not make the user wait, but here you are using only `deliver`. wouldn't it make sense to use the same method for testing?

Reply:
nope, because if I use deliver_later in the test I cannot know if and when the email will be delivered. And as I want to know what the mail is like I want to get it delivered now. This isn't about testing deliver_later or deliver. Its for my own funcionality

Reply:
aaaah 💡  yes makes sense